setup.cfg

```diff
@@ -21,6 +21,7 @@ install_requires =
     configobj
     hidapi
     inotify
```

Comment:
This one can go now, right? I guess I simply mixed up the packages.

Reply:
Yeah, I suppose it can actually. I'll update that